bridge/engine/SsrcOutboundContext.h

```diff
@@ -86,6 +86,25 @@ class SsrcOutboundContext
     uint64_t lastSendTime;
     bool markedForDeletion;
     bool idle;
+
+    struct PliMetric
+    {
+        uint64_t userRequestTimestamp = 0;
+        uint64_t triggerTimestamp = 0;
+        uint64_t keyFrameTimestamp = 0;
+
+        uint64_t getDelay(const uint64_t timestamp) const
+        {
+            if (userRequestTimestamp != 0 && static_cast<int64_t>(userRequestTimestamp - keyFrameTimestamp) > 0)
```

Comment:
What if have both? (triggered and user-requested?)

Reply:
yes, then we log the user requested one